examples/terminal-attributes.php

```diff
@@ -79,21 +77,35 @@ function () use ($xterm) {
         [11, 40],
     ];
 
+    $xterm
+        ->bgBlue()
+        ->brightYellow()
+        ->eraseAll();
+
     $i = 0;
     foreach ($view as $title => $data) {
         $element = str_pad($title . ' ', 25, '.') . ' : ' . $data;
         $xterm
             ->moveCursorTo(...$lines[$i++])
             ->write($element);
     }
-        //
+    //
     $xterm
         ->moveCursorTo(15, 10)
         ->blink()
         ->write("Press any key to exit or resize/move your screen")
         ->normal()
         ->flush()
     ;
+};
+
+async($update);
+async(function () use (&$update) {
+    delay(1);
+    if ($update !== null) {
+        $update();
+    }
 });
```

Comment:
Wonder why this is needed to be like this (and queue twice). I think maybe there is something wrong with the way the reporting functions work (for example https://github.com/Aspectus-PHP/terminal/blob/improve-terminal-attributes/src/Xterm.php#L249)